rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPNodeGroupRepository.scala

```diff
@@ -757,8 +747,10 @@ class WoLDAPNodeGroupRepository(
   override def create(nodeGroup: NodeGroup, into: NodeGroupCategoryId, modId: ModificationId, actor:EventActor, reason:Option[String]): Box[AddNodeGroupDiff] = {
     for {
       con           <- ldap
-      exists        <- if (nodeGroupExists(con, nodeGroup.name)) Failure("Cannot create a group with name %s : there is already a group with the same name".format(nodeGroup.name))
-                       else Full(Unit)
+      exists        <- if (checkNodeGroupExists(con, nodeGroup))
+                         Failure(s"Cannot create a group '${nodeGroup.name}': a group with the same id (${nodeGroup.id.value}) or name already exists".format(nodeGroup.name))
```

Comment:
You're right, format is the old way to build string with values, i removed "format" calls from ost string but forget at least one ... thanks !